cmd/neofs-node/object.go

```diff
@@ -314,6 +317,7 @@ func initObjectService(c *cfg) {
 		keys:    keyStorage,
 	}
 	server := objectService.New(objSvc, mNumber, fsChain, storage, c.metaService, c.shared.basics.key.PrivateKey, c.metricsCollector, aclChecker, aclSvc, coreConstructor)
+	os.server = server
```

Comment:
Seems racy, `os` is already given to the `tombstone.NewVerifier()`. Not sure there are any threads involved though, maybe not a problem.